presto-iceberg/src/test/java/com/facebook/presto/iceberg/rest/TestIcebergSmokeRest.java

```diff
@@ -206,4 +206,10 @@ public void testSetOauth2ServerUriPropertyI()
 
         assertEquals(catalog.properties().get(OAUTH2_SERVER_URI), authEndpoint);
     }
+
+    @Override
+    public void testDeprecatedTablePropertiesCreateTable()
+    {
+        // fails due to bug in earlier iceberg versions. Remove when iceberg library is updated
```

Comment:
What was the error? Could you please add a little more information about the error here?

Reply:
It's a bug in the RestCatalog implementation which makes it impossible to create a table with `format-version` set to 1.

Reply:
I updated the comment